docs/setup/glue.md

```diff
@@ -0,0 +1,114 @@
+
+This tutorial will cover how to configure both a glue notebook and a glue ETL job. The tutorial is written assuming you
+have a working knowledge of AWS Glue jobs and S3.
+
+In the tutorial, we use
+Sedona 1.6.0 and [Glue 4](https://docs.aws.amazon.com/glue/latest/dg/release-notes.html) which runs on Spark 3.3.0, Java 8, Scala 2.12,
+and Python 3.10. We recommend Sedona-1.3.1-incubating and above for Glue.
+
+## Stage Sedona Jar in S3
+
+In an AWS S3 bucket you will need the sedona-spark-shaded and geotools-wrapper jars. There are two options to get these.
+Ensure the locations of the jars are accessible to your glue job.
+
+### Option 1: Use the Wherobots-hosted jars
+
+[Wherobots](https://wherobots.com/) provides a public S3 bucket with the necessary jars. You can point to these directly in your glue jobs'
+configurations. For 1.6.0, the sedona and geotools jars are available at the following locations:
+
+* `s3://wherobots-sedona-jars/1.6.0/sedona-spark-shaded-3.0_2.12-1.6.0.jar`
```

Comment:
Can you use the variable {{ sedona.current_version }} instead of hard coded version. See the example in other tutorial.

Reply:
how can we ensure that these are updated when a new version of sedona is released?

Reply:
Because I will update the variable name after each release: https://github.com/apache/sedona/blob/master/mkdocs.yml#L161

Reply:
Sorry, I mean the jars in the s3 bucket.

Reply:
Sadly, someone needs to add it manually.

Reply:
Is there a release checklist or something to which we can add to ping me to update the bucket?

Reply:
Checklist is here: https://sedona.apache.org/1.6.0/community/publish/